crypto/mem_dbg.c

```diff
@@ -598,9 +599,10 @@ static void print_leak(const MEM *m, MEM_LEAK *l)
 
             ami_cnt++;
             memset(buf, '>', ami_cnt);
+            tid.tid = amip->threadid;
```

Comment:
I thought about that but AFAIK we don't need to do this anywhere else, and there isn't a great way to do this. This is ok for this scenario but maybe not for others. Probably best to keep as is.